### PR TITLE
Change name of flag from with_armature to without_armature

### PR DESCRIPTION
**Pull request by andreadelprete**
As suggested by @cmastalli in #936 I have changed the name of the flag `with_armature` to `without_armature`.

### CODE REVIEW

**Review by cmastalli — APPROVED** [review]
Thanks @andreadelprete  ;)

**Comment by andreadelprete** [comment]
I cannot merge as I don't have write access

**Pull request merged**
